crates/swc_ecma_utils/Cargo.toml

```diff
@@ -35,4 +35,5 @@ swc_ecma_visit = { version = "0.98.7", path = "../swc_ecma_visit" }
 
 [dev-dependencies]
 swc_ecma_parser = { version = "0.143.10", path = "../swc_ecma_parser" }
+swc_ecma_transforms_base = { version = "0.137.16", path = "../swc_ecma_transforms_base" }
```

Comment:
You should move tests instead

Reply:
i'm a little confused now : (
i wonder whether the top level  assert test case is necessary; because resolver is working fine now, and it has no business with the export statement.

Reply:
I mean, one cannot publish a crate if there's a cyclic dependency between crates.
Please revert this dependency, not test

Reply:
got it.
i just remove the test  : p